### PR TITLE
working migration

### PR DESCRIPTION
**Pull request by dime-squanderer**

### CODE REVIEW

File: lib/data/sources/local/db.dart [hunk 1]
```diff
@@ -100,6 +100,29 @@ class DB extends _$DB {
...
+                    name TEXT NOT NULL DEFAULT '',
+                    encrypted_private_key TEXT NOT NULL UNIQUE,
+                    wallet_uuid TEXT NOT NULL,
+                    PRIMARY KEY (address)
+                  );
+
+                  -- Copy data from old table to new table
+                  INSERT INTO imported_addresses_temp (address, encrypted_private_key, wallet_uuid)
+                  SELECT address, encrypted_private_key, wallet_uuid
+                  FROM imported_addresses;
```
Comment (dime-squanderer) [inline lib/data/sources/local/db.dart:117]: this is unnecessary since there is no data in the table, but better to be safe I think.

Comment (mattmarcello) [reply lib/data/sources/local/db.dart:117]: are you storing the private key or the WIF?

Comment (mattmarcello) [reply lib/data/sources/local/db.dart:117]: there is no old table, right? 

I think it's better just to be consistent here.  this leaves us with the impression that we are modifying an existing table as opposed to creating a new one.

Comment (mattmarcello) [reply lib/data/sources/local/db.dart:117]: ( i know that's what you're saying in your comment ) :)

**Review by mattmarcello — APPROVED** [review]

File: lib/data/sources/local/db.dart (hunk 1 above)
Comment (dime-squanderer) [reply lib/data/sources/local/db.dart:117]: there is an old table, but it has no data. also, we are storing the pk wif

**Pull request merged**